server/src/clientglue.js

```diff
@@ -1,4 +1,5 @@
-/* jshint browser:true */
+/* jslint browser:true */
+/* global POST_MESSAGE_ORIGIN */
```

Comment:
This might be clearer as `CONTENT_HOSTING_ORIGIN` (I was going to say `CONTENT_ORIGIN` but that feels a bit like the source URL origin)